# -*- coding: utf-8 -*-

from django.core import exceptions

class UserCannotFollowOrgTwice(exceptions.PermissionDenied):

    def __init__(self, user, org):
        self.user = user
        self.org = org

    def __unicode__(self):
        return u"L'utente %s sta gi√† seguendo l'associazione %s" % (self.user, 
        self.org)
